test/ASTGen/attrs.swift

```diff
@@ -12,9 +12,8 @@
 
 // REQUIRES: executable_test
 // REQUIRES: swift_swift_parser
+// REQUIRES: swift-feature-ParserASTGen
```

Comment:
All existing features that can be tested with `REQUIRES:` are snake case. Can you make the feature ones snake case instead of kebab case too?

Reply:
Do you mean `swift_feature_ParserASTGen`? I would like not to transform the feature names and keep those as the original ones in `Features.def` (camel case).

(Never heard "kebab case", but it is a great name)

Reply:
Yes, I'm suggesting `swift_feature_ParserASTGen` instead of `swift-feature-ParserASTGen`. I agree it's not worthwhile to try to transform the feature identifiers themselves.